source/_posts/2020-03-14-2020-using-vyper-with-embark.md

```diff
@@ -0,0 +1,204 @@
+title: Using Vyper with Embark
+summary: "Vyper has gained some traction on Solidity as an Ethereum smart contract language, especially with support going in to ETH2. Learn how to use Embark to use Vyper smart contracts in a &ETH;App."
```

Comment:
Is the `&ETH;App` intended here?

Reply:
It creates the cool D like so: Ð

Reply:
TIL!

Also funny that the encoding is &ETH; haha